concepticondata/conceptlists/Seifart-2015-414.tsv

```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-86	86	paca	316	PACA	110
+Seifart-2015-414-87	87	crocodile	1857	CROCODILE	111
+Seifart-2015-414-88	88	iguana	1578	IGUANA	112
+Seifart-2015-414-89	89	collared peccary	435	COLLARED PECCARY	115
+Seifart-2015-414-90	90	white-lipped peccary	436	WHITE-LIPPED PECCARY	116
+Seifart-2015-414-91	91	agouti	313	AGOUTI	117
+Seifart-2015-414-92	92	rat	1490	RAT	118
+Seifart-2015-414-93	93	cat	1208	CAT	119
+Seifart-2015-414-94	94	mouse	1491	MOUSE	120
+Seifart-2015-414-95	95	tail	1220	TAIL	121
+Seifart-2015-414-96	96	snake	730	SNAKE	122
+Seifart-2015-414-97	97	anaconda,water boa	896	ANACONDA (WATER BOA)	123
+Seifart-2015-414-98	98	rattlesnake	871	RATTLESNAKE	124
+Seifart-2015-414-99	99	coral snake	2240	CORAL SNAKE	125
+Seifart-2015-414-100	100	toad	894	TOAD	126
+Seifart-2015-414-101	101	hummingbird	1589	HUMMINGBIRD	128
+Seifart-2015-414-102	102	macaw	1630	MACAW	129
+Seifart-2015-414-103	103	toucan	308	TOUCAN	130
+Seifart-2015-414-104	104	buzzard	1865	BUZZARD	133
+Seifart-2015-414-105	105	curassow	315	CURASSOW	134
+Seifart-2015-414-106	106	guan	437	GUAN	136
+Seifart-2015-414-107	107	hen	1514	HEN	137
+Seifart-2015-414-108	108	piranha	317	PIRANHA	139
+Seifart-2015-414-109	109	flea	232	FLEA	142
+Seifart-2015-414-110	110	louse	1392	LOUSE	143
+Seifart-2015-414-111	111	mosquito	1509	MOSQUITO	144
+Seifart-2015-414-112	112	termite	883	TERMITE	145
+Seifart-2015-414-113	113	ant	587	ANT	146
+Seifart-2015-414-114	114	spider	843	SPIDER	147
+Seifart-2015-414-115	115	jigger (tunga penetrans)	438	CHIGGOE (JIGGER FLEA)	148
+Seifart-2015-414-116	116	bush	1683	BUSH	149
+Seifart-2015-414-117	117	open grassland	260	SAVANNA	150
+Seifart-2015-414-118	118	hill	614	HILL	151
+Seifart-2015-414-119	119	tree	906	TREE	152
+Seifart-2015-414-120	120	leaf	628	LEAF	153
+Seifart-2015-414-121	121	tree leaf	628	LEAF	154
+Seifart-2015-414-122	122	root	670	ROOT	158
+Seifart-2015-414-123	123	seed	714	SEED	159
+Seifart-2015-414-124	124	stick	1295	STICK	160
+Seifart-2015-414-125	125	grass	606	GRASS	161
+Seifart-2015-414-126	126	cassava (bread)	925	CASSAVA	163
+Seifart-2015-414-127	127	tobacco	974	TOBACCO	165
+Seifart-2015-414-128	128	cotton	1850	COTTON	166
+Seifart-2015-414-129	129	gourd	411	GOURD	167
+Seifart-2015-414-130	130	yam	410	YAM	168
+Seifart-2015-414-131	131	annatto	349	ACHIOTE	170
+Seifart-2015-414-132	132	chili pepper	919	CHILI PEPPER	171
+Seifart-2015-414-133	133	coca	137	COCA	172
+Seifart-2015-414-134	134	banisterium	276	BANISTERIUM	173
+Seifart-2015-414-135	135	plantain	399	PLANTAIN	174
+Seifart-2015-414-136	136	palm	1181	PALM TREE	175
+Seifart-2015-414-137	137	salt	1274	SALT	179
+Seifart-2015-414-138	138	chicha (of peach palm fruits)	441	CHICHA	180
+Seifart-2015-414-139	139	one	1493	ONE	181
+Seifart-2015-414-140	140	two	1498	TWO	182
+Seifart-2015-414-141	141	five	493	FIVE	185
+Seifart-2015-414-142	142	first	728	FIRST	191
+Seifart-2015-414-143	143	last	1529	LAST (FINAL)	192
+Seifart-2015-414-144	144	drum	908	DRUM	194
+Seifart-2015-414-145	145	ear ornament			196
```

Comment:
There are many entries in Concepticon regarding "ear ornament". Please open an issue whether those should be mapped under "earring", or whether we should add a new concept "ear ornament".

Reply:
This was already responded in #1336. We should map this gloss as EARRING. With the last commit, this was implemented.